lib/rubocop/cli.rb

```diff
@@ -38,6 +38,9 @@ def run(args = ARGV)
       return 2
     rescue Finished
       return 0
+    rescue ArgumentError => e
```

Comment:
I think `ArgumentError` is too general. Probably it conceals other `ArgumentError`.
So I think `validate_cop_list` method should raise an error with more specific error class, and it is caught at here.

Reply:
Certainly, I think you are quite right. I think I will make original exception `IncorrectCopNameError`.